build/drivers/driver-installer.nsi

```diff
@@ -0,0 +1,56 @@
+;-------
+;General
+
+;Name of the installer.
+Name "Driver installer"
+
+;Name of the .exe file created.
+OutFile "driver-installer.exe"
```

Comment:
The name here is perhaps a bit generic? How about something like "nordic-usb-driver-installer.exe"?